CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 ## Changelog
 ### next
+  * tpm2_nvwrite: short option changed to argument
+    -x used to specify nv index is now an argument
+  * tpm2_nvwrite: input file changed from argument to option
+    -i or --input is now the method to specify file data to write
+  * tpm2_nvlist: Renaming the tool
+    - tpm2_nvlist is now tpm2_nvreadpublic
+  * tpm2_nvundefine: short option changed to argument
+    -x used to specify nv index is now an argument
+  * tpm2_nvrelease: Renaming the tool
+    - tpm2_nvrelease is now tpm2_nvundefine
```

Comment:
Are you going to use tpm2_nvundefinespace instead of tpm2_nvundefine?
If TPM2_NV_UndefineSpaceSpecial implemented, what is the suggest name?

Reply:
I would do it as an option or read the attributes of the nv index and figure out which one to call. Similar to how changeauth works or create/createloaded.